lib/synced/strategies/full.rb

```diff
@@ -192,6 +192,15 @@ def search_params
         end]
       end
 
+      def scope_params
+        case @scope
+        when Rental
```

Comment:
We could make it more generic, maybe add module `Synced::ActsAsScope` that would define such behavior, here we would simply call `@scope.scope_params` and by default it would return empty hash.